content/en/agent/faq/install-the-agent-with-aws-ssm.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Install the Agent with AWS SSM
+kind: faq
+further_reading:
+- link: "agent/"
+  tag: "Documentation"
+  text: "Learn more about the Datadog Agent"
+---
+
+Follow the steps below to install the Datadog Agent on an EC2 instance with AWS Systems Manager.
```

Comment:
Might add a note about this being an "example configuration for a single manual instance being deployed". Beyond launching the instance, the rest of the steps `should` apply to autoscaling infrastructure, etc...